tests/unit/helpers/ember-prop-types-test.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * Unit test for the frost-power-display component
+ */
+
+// Because we want to satisfy ESLint rules for not using references that were not previously defined...
+// ...AND... we need to be able to spy on the same methods we need to import the "mocha" library twice,
+// which is why the second import of it disables the ESLint warning about such duplication on a per-line basis
+import {expect} from 'chai'
+import Ember from 'ember'
+import {afterEach, beforeEach, describe, it} from 'mocha'
+import * as spyOnMocha from 'mocha' // eslint-disable-line no-duplicate-imports
+import sinon from 'sinon'
+import validatePropTypes, * as PropTypesHelper from 'dummy/tests/helpers/ember-prop-types'
```

Comment:
Is there a reason we're wrapping the `forTestingOnly` in `PropTypesHelper` here? Seems like it would be simpler to just have:

```js
import validatePropTypes, {forTestingOnly} from 'dummy/tests/helpers/ember-prop-types'
```
Then you could remove all the `PropTypesHelper` references below.